win32ss/printing/base/spoolss/tools.c

```diff
@@ -49,7 +49,22 @@ PackStrings(PWSTR* pSource, PBYTE pDest, const DWORD* DestOffsets, PBYTE pEnd)
         if (*pSource)
         {
             // Determine the length of the source string.
-            cbString = (wcslen(*pSource) + 1) * sizeof(WCHAR);
+            if (*DestOffsets & MARSHAL_MULTI_SZ)
```

Comment:
I didn't know of that flag. If it's also supported in the Windows original, please add a test/example to modules/rostests/apitests/spoolss/PackStrings.c before merging this PR.

Reply:
I made that flag up on the spot, I didnt realize that PackStrings was exported somewhere....
Guess I have to come up with another way of packing a multi-sz string :(
Maybe I can leave this field empty for now.